client/me/purchases/purchases-list-in-dataviews/index.tsx

```diff
@@ -0,0 +1,205 @@
+import config from '@automattic/calypso-config';
+import { Card } from '@automattic/components';
+import { Purchases, SiteDetails } from '@automattic/data-stores';
+import { isValueTruthy } from '@automattic/wpcom-checkout';
+import { LocalizeProps, localize } from 'i18n-calypso';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import QueryConciergeInitial from 'calypso/components/data/query-concierge-initial';
+import QueryMembershipsSubscriptions from 'calypso/components/data/query-memberships-subscriptions';
+import QueryUserPurchases from 'calypso/components/data/query-user-purchases';
+import { DataViews } from 'calypso/components/dataviews';
+import NoSitesMessage from 'calypso/components/empty-content/no-sites-message';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import Main from 'calypso/components/main';
+import NavigationHeader from 'calypso/components/navigation-header';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { getPurchasesBySite, getSubscriptionsBySite } from 'calypso/lib/purchases';
+import { MembershipSubscription, Purchase } from 'calypso/lib/purchases/types';
+import { PurchaseListConciergeBanner } from 'calypso/me/purchases/purchases-list/purchase-list-concierge-banner';
+import PurchasesNavigation from 'calypso/me/purchases/purchases-navigation';
+import titles from 'calypso/me/purchases/titles';
+import {
+	WithStoredPaymentMethodsProps,
+	withStoredPaymentMethods,
+} from 'calypso/my-sites/checkout/src/hooks/use-stored-payment-methods';
+import { getAllSubscriptions } from 'calypso/state/memberships/subscriptions/selectors';
+import {
+	getUserPurchases,
+	hasLoadedUserPurchasesFromServer,
+	isFetchingUserPurchases,
+} from 'calypso/state/purchases/selectors';
+import getAvailableConciergeSessions from 'calypso/state/selectors/get-available-concierge-sessions';
+import getConciergeNextAppointment, {
+	NextAppointment,
+} from 'calypso/state/selectors/get-concierge-next-appointment';
+import getConciergeUserBlocked from 'calypso/state/selectors/get-concierge-user-blocked';
+import getSites from 'calypso/state/selectors/get-sites';
+import { getSiteId } from 'calypso/state/sites/selectors';
+import { AppState } from 'calypso/types';
+import MembershipSite from '../membership-site';
+import PurchasesSite from '../purchases-site';
+import { purchasesDataFields } from './purchases-data-field';
+import { purchasesDataView } from './purchases-data-view'; // This is the temporary data
```

Comment:
Can you explain more what makes this temporary data? I guess because the view will eventually be dynamic state? If so, then maybe this data could just be inline in this file to avoid needing to add an additional file just for something you're planning to remove.

Reply:
This was an old comment. I plan to keep the file around, originally the DataView file was pushing the "faked" data I had used in my POC. It is probably more accurate to say that this is a WIP since the view will be updated as we pull more columns out from the PurchaseItem component.

I removed the comment for now.